cms/djangoapps/contentstore/tasks.py

```diff
@@ -232,6 +232,31 @@ def update_library_index(library_id, triggered_time_isoformat):
         LOGGER.debug('Search indexing successful for library %s', library_id)
 
 
+@shared_task
+@set_code_owner_attribute
+def update_special_exams_and_publish(course_key_str):
+    """
+    Registers special exams for a given course and calls publishing flow.
+
+    on_course_publish expects that the edx-proctoring subsystem has been refreshed
+    before being executed, so both functions are called here synchronously.
+    """
+    from cms.djangoapps.contentstore.proctoring import register_special_exams
+    from openedx.core.djangoapps.credit.signals import on_course_publish
+
+    course_key = CourseKey.from_string(course_key_str)
+    LOGGER.debug('Attempting to register exams for course %s', course_key_str)
```

Comment:
Instead of debug, make these info for now? When this async action is rolled out, then we can choose to set them to debug, or delete those not that useful.